BraintreeUIKit/Components/BTUIKCardListLabel.m

```diff
@@ -40,7 +40,7 @@ - (UIImage *) imageWithView:(UIView *)view
 }
 
 - (void)setAvailablePaymentOptions:(NSArray *)availablePaymentOptions {
-    _availablePaymentOptions = availablePaymentOptions;
+    _availablePaymentOptions = [[NSSet setWithArray:availablePaymentOptions] allObjects];
```

Comment:
Is there any concern we should unique the values when loading the configuration itself instead of only on this class for display?

Reply:
That's a good question -- I'd be inclined to make the change at a higher level within Drop-in, but if we go with the assumption that we'll patch this on the server, I think this is an acceptable stopgap.